packages/api/internal/handlers/sanbox_metrics.go

```diff
@@ -34,19 +34,21 @@ func (a *APIStore) GetSandboxesSandboxIDMetrics(
 		attribute.String("team.id", teamID.String()),
 	)
 
-	var start time.Time
-
 	end := time.Now()
+	start := end.Add(-oldestLogsLimit)
 
 	// Sanitize ID
 	// https://grafana.com/blog/2021/01/05/how-to-escape-special-characters-with-lokis-logql/
 	id := strings.ReplaceAll(sandboxID, "`", "")
+
+	// equivalent CLI query:
+	// logcli query '{source="logs-collector", service="envd", teamID="65d165ab-69f6-4b5c-9165-6b93cd341503", sandboxID="izuhqjlfabd8ataeixrtl", category="metrics"}' --from="2025-01-19T10:00:00Z"
 	query := fmt.Sprintf(
-		"{source=\"logs-collector\", category=\"metrics\", service=\"envd\", teamID=`%s`, sandboxID=`%s`}", teamID.String(), id)
```

Comment:
There's reason for `, you don't have to escape such strings https://grafana.com/blog/2021/01/05/how-to-escape-special-characters-with-lokis-logql/